examples/nat_wiring.json

```diff
@@ -7,6 +7,7 @@
         "next_outside_hop_mac" : "00:00:00:00:00:09",
         "tcp_start_port" : "35000",
         "udp_start_port" : "35000",
+        "tcp_time_wait" : "00:00:05",
```

Comment:
Out of interest have you tried experimenting with larger timeouts and large numbers of connections? It would be interesting to see how the system's behaviour changes (degrades) under load.